### ISSUE TITLE
validator warnings: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.

### ISSUE THREAD

**Issue by nschonni** [w3c/aria-practices#1678]
Affected files:
```
"file:/mnt/z/aria-practices/aria-practices.html":13.3-13.25: info warning: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.
"file:/mnt/z/aria-practices/aria-practices.html":22.3-22.21: info warning: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.
"file:/mnt/z/aria-practices/examples/feed/feedDisplay.html":20.5-20.31: info warning: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.
"file:/mnt/z/aria-practices/examples/feed/feedDisplay.html":24.5-24.28: info warning: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.
```

**Comment by carmacleod** [comment]
> "file:/mnt/z/aria-practices/aria-practices.html":13.3-13.25: info warning: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.

Respec adds the "Abstract" heading (h2) because the section's id="abstract".

> "file:/mnt/z/aria-practices/aria-practices.html":22.3-22.21: info warning: Section lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all sections.

Respec adds the "Status of This Document" heading (h2) because the section's id="sotd".

These 2 places are examples where it would be great if we could run the validator on the generated page, rather than on the html source code.

**Comment by mcking65** [comment]
Those two should go away with the redesign because we won't be using respec ... or at least I think they will go away. I hope we don't recreat this problem in another form.
@jesdaigle, please take note.

**Comment by jesdaigle** [comment]
@mcking65, noted.
